components/Header.tsx

```diff
@@ -11,24 +11,26 @@ import Search, { SearchProps } from './Search';
 const SearchBar = (props: SearchProps) => {
   const router = useRouter();
 
-  return (router.pathname === '/repos/[language]' && (
+  return (router.pathname === '/repos/[language]' ?(
       <Search {...props} />
     )
+    : 
+     <div className={props as string}></div>
```

Comment:
Instead of directly putting an object `as string` it's better to maybe put nothing or take `className` as a prop